### PR TITLE
Splits Adrenaline Nanites In Two Separate Programs

### PR DESCRIPTION
**Pull request by eeSPee**
## About The Pull Request

Splits Adrenaline Nanites into Adrenaline Nanites and Amphetamine Burst; one removes your stuns, another gives you movement speed. Amphetamine burst nanites are hidden behind Illegal Tech.

## Why It's Good For The Game

Really feels like Adrenaline Nanites were grudge-nerfed. I don't know if the nerf needed to be this harsh.

## Changelog
:cl:
tweak: Splits Adrenaline Nanites into Adrenaline Nanites and Amphetamine Burst
balance: Adjusted costs of Adrenaline Nanites to compensate for the nerf
/:cl:

### CODE REVIEW

**Comment by PowerfulBacon** [comment]
Nanites to remove all stuns being easilly accessible is a bad idea imo

**Comment by Cenrus** [comment]
Amphetamine is an antag only chem, traitors get three uses of it in their uplink for 8TC. That's almost half their syndi bucks for a limited use thing. Nanites recharge over time for free.

**Comment by eeSPee** [comment]
> Amphetamine is an antag only chem, traitors get three uses of it in their uplink for 8TC. That's almost half their syndi bucks for a limited use thing. Nanites recharge over time for free.

Amph nanites require illegal tech and are an underwhelming version of the old adrenaline nanites.

> Nanites to remove all stuns being easilly accessible is a bad idea imo

They have really long cooldown and only work once. Maybe they will make the '2 stamina hits = crit' meta less prevalent.

**Comment by Cenrus** [comment]
If I'm seeing this correctly you're putting the Haste nanites (Amphetamine) under the biological nanites node which doesn't need illegal tech to get.

**Comment by eeSPee** [comment]
Oversight; thanks for pointing it out!

File: code/modules/research/nanites/nanite_programs/buffing.dm [hunk 1]
```diff
@@ -22,7 +22,7 @@
 	name = "Adrenaline Burst"
 	desc = "The nanites cause a burst of adrenaline when triggered, allowing the user to push their body past its normal limits."
 	can_trigger = TRUE
-	trigger_cost = 25
+	trigger_cost = 20
```
Comment (bloons3) [inline code/modules/research/nanites/nanite_programs/buffing.dm:25]: undocumented change

**Comment by OceanFish1** [comment]
Day 10 of trying hopelessly to revive adrenals from the state they have gone to

**Review by utilbird — DISMISSED** [review]

**Review by utilbird — APPROVED** [review]

**Pull request merged**